frontend/src/css/navbar.css

```diff
@@ -15,14 +17,25 @@
 }
 
 .nav-container {
-  width: 20%;
+  position: relative;
+  width: 15%;
   height: 100%;
-  margin-top: 1%;
+  margin-top: 0.7%;
+  margin-left: 0.4%;
+  align-self: flex-start;
+}
+
+.picker-container {
+  position: relative;
+  width: 15%;
```

Comment:
so apparently the bug where the dates picker go outside of the navbar could be because of the picker container.    I tried to put "fit-content" and the bug disappears (although we should fix the others button clashing with each other) 
<img width="547" alt="image" src="https://github.com/DTU-SE-Group-D/LiRA-Viz/assets/108626122/42d76299-9459-4ef3-b8a9-319fccf987c8">

also, there is a drawback:  with fit-content the month selection becomes this ahaha

<img width="1213" alt="image" src="https://github.com/DTU-SE-Group-D/LiRA-Viz/assets/108626122/ae0a85e8-f71a-4125-a23f-8f78491c27f5">


we could put the width as a fixed size?  like 250px?

Reply:
and even when resizing the window we the dropdown parameters values should be readable I imagine (?).